### PR TITLE
docs: Added installation guide for RHEL/Feroda

### LINKED ISSUE #1051: Add installation guide for RHEL/Fedora
## Description

The installation guide currently has instructions for Ubuntu. Add the steps to be followed while setting up VMS in RHEL/Fedora to the [installation guide](https://github.com/anitab-org/vms/blob/develop/aut_docs/Installation_Setup.md).

### Update [Required]
- [ ] A section in the installation guide to setup the project in RHEL/Fedora.

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
2 hours

### PR DESCRIPTION
**Pull request by khushishikhu**
# Description
Installation guide for RHEL/Feroda

Fixes #1051 

# Type of Change:

- Code
- Quality Assurance
- Documentation

# How Has This Been Tested?
Describe the tests you ran to verify your changes. Provide instructions or GIFs so we can reproduce. List any relevant details for your test.

# Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged 

**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules

### CODE REVIEW

**Comment by khushishikhu** [comment]
please someone review my PR and let me know what other changes i need to do to make it better.

**Review by rkpattnaik780 — COMMENTED** [review]
These commands are good enough for installing in RHEL 8. @satya7289 I need some thoughts about this PR. Since installation for Fedora and Ubuntu differ in only two sections, do you think there is a need for re-writing the rest of the sections?

**Comment by rkpattnaik780** [comment]
@khushishikhu since most of the sections dont depend on platform, can we just add the specific sections and not all of them. Maybe we can just have a "one-time setup in RHEL/Fedora" just below the existing "A one-time setup" and do something similar for "Testing session".

**Comment by khushishikhu** [comment]
@rkpattnaik780  okay, i will update you with those changes

**Review by rkpattnaik780 — COMMENTED** [review]
LGTM! Please squash the commits so that I can approve.

**Comment by khushishikhu** [comment]
> LGTM! Please squash the commits so that I can approve.

can you please explain me how to squash my commits.

**Comment by khushishikhu** [comment]
@rkpattnaik780  i don't know why but i am not able to squash the commits

**Comment by rkpattnaik780** [comment]
> @rkpattnaik780 i don't know why but i am not able to squash the commits

You can refer [this article](https://medium.com/@slamflipstrom/a-beginners-guide-to-squashing-commits-with-git-rebase-8185cf6e62ec), if your problem is not being covered here, please let me know in this thread, attach screenshots if needed.

**Comment by khushishikhu** [comment]
> > @rkpattnaik780 i don't know why but i am not able to squash the commits
> 
> You can refer [this article](https://medium.com/@slamflipstrom/a-beginners-guide-to-squashing-commits-with-git-rebase-8185cf6e62ec), if your problem is not being covered here, please let me know in this thread, attach screenshots if needed.

actually it's not showing my commits in my branch whenever i run `git rebase -i HEAD~3`

**Comment by rkpattnaik780** [comment]
> > > @rkpattnaik780 i don't know why but i am not able to squash the commits
> > 
> > 
> > You can refer [this article](https://medium.com/@slamflipstrom/a-beginners-guide-to-squashing-commits-with-git-rebase-8185cf6e62ec), if your problem is not being covered here, please let me know in this thread, attach screenshots if needed.
> 
> actually it's not showing my commits in my branch whenever i run `git rebase -i HEAD~3`
> gi

Seems you are in the develop branch. Can you confirm that you are in branch `patch-1-1` by checking the output of `git branch` and then check your commit history through `git log`.

**Comment by khushishikhu** [comment]
gi
in commit history it's not showing me my commits

**Comment by rkpattnaik780** [comment]
I would suggest to make a new commit. That would need you to reset the current changes and copy paste from the files already pushed to github. Try to do the following steps:
1. git fetch upstream
2. git reset --hard HEAD~10
3. git rebase upstream/develop
(These steps will undo all your changes).
4. Copy paste contents of [this file](https://raw.githubusercontent.com/anitab-org/vms/6bcff9d17c18ace85c1b79723bf241c562ade5ba/aut_docs/Installation_Setup.md) to local file.
5. Commit the changes as you usually do.
6. git push -f origin/patch-1-1

Hopefully these will get the work done.

**Comment by khushishikhu** [comment]
@rkpattnaik780  actually somehow i got to know by myself that why my commits were not squashing. thank you

**Review by rkpattnaik780 — DISMISSED** [review]
LGTM!

**Comment by aniketrb-github** [comment]
> @rkpattnaik780 actually somehow I got to know by myself that why my commits were not squashing. thank you

Hi @rkpattnaik780 and @khushishikhu.
I'm just leaving these [two ](https://youtu.be/2E23I9PzplM ) [links ](https://youtu.be/V5KrD7CmO4o) here which I found were very easy to understand, learn & execute with necessary pre-requisites regarding commit squash.
I just learned the commit squash concept yesterday & did struggle a little bit...but later got over it due to the above share links.
Thanks. :)

**Comment by khushishikhu** [comment]
@rkpattnaik780  now can i just go for other issues to work for OSH

**Comment by rkpattnaik780** [comment]
> @rkpattnaik780 now can i just go for other issues to work for OSH

Go ahead.

File: aut_docs/Installation_Setup.md [hunk 1]
```diff
@@ -25,6 +25,27 @@
...
+    ```
+
+- Install virtual environment
+    ```bash
+      sudo yum install python-virtualenv
+    ```
+
+- Clone the VMS project
+    ```bash
+      git clone https://github.com/systers/vms.git
```
Comment (isabelcosta) [inline aut_docs/Installation_Setup.md:42]:
```suggestion
      git clone https://github.com/anitab-org/vms.git
```

**Comment by khushishikhu** [comment]
Thank you @isabelcosta  for reviewing my PR . I have done the required changes

**Comment by khushishikhu** [comment]
@rkpattnaik780 please review my PR

**Comment by rkpattnaik780** [comment]
Squash the commits.

**Comment by khushishikhu** [comment]
@rkpattnaik780  i have squashed the commits

**Review by rkpattnaik780 — APPROVED** [review]

**Review by isabelcosta — APPROVED** [review]

**Pull request merged**